src/js/queries/getOfficialDocumentPageQuery.js

```diff
@@ -24,6 +24,7 @@ const getOfficialDocumentPageQuery = `
                 document {
                   fileSize
                   filename
+                  url
                 }
               }
             }
```

Comment:
is this query being used anywhere?

Reply:
No, but I wasn't sure if you were going to use it or not for your stuff.